### ISSUE TITLE
Adding Light / Dark Mode

### ISSUE THREAD

**Issue by Frostfire25** [Merrimack-Computer-Club/Merrimack_Computer_Club_Website#37]

**Comment by Frostfire25** [comment]
https://v1.mantine.dev/guides/dark-theme/

**Comment by zablon-oigo** [comment]
Hey, Is the issue still open? , I would like to give it a try, please explain how you would like this to be done

**Comment by Frostfire25** [comment]
> Hey, Is the issue still open? , I would like to give it a try, please explain how you would like this to be done

Yes, it is! Could you add me on discord (alexelg) or send a message in the MCC Discord chat-> #mcc-website.
Then I can get you coordinated with this repository.
